packages/transactions/src/App.tsx

```diff
@@ -1,10 +1,16 @@
 import React, { FC } from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
 import { LoadingApp } from '@openmsupply-client/common';
+import TransactionService from './TransactionService';
+
+const queryClient = new QueryClient();
 
 const App: FC = () => (
-  <React.Suspense fallback={<LoadingApp />}>
-    <span>Transactions</span>
-  </React.Suspense>
+  <QueryClientProvider client={queryClient}>
```

Comment:
Does the provider have to be in here as well as in the host?

Reply:
Yeah. This is the `Transaction` MFE `App` which will load when navigating to `localhost:3005` - it doesn't have the host but does have HMR. The `TransactionService` is what is exposed as a remote.